lib/say_when/storage/memory/job.rb

```diff
@@ -17,21 +17,45 @@ class Job
         has_properties :trigger_strategy, :trigger_options, :last_fire_at, :next_fire_at
         has_properties :job_class, :job_method, :data
         has_properties :scheduled
+        has_properties :updated_at
+
+        def self.class_lock
+          @@_lock ||= Mutex.new
+        end
+
+        def self._reset
+          @@jobs = SortedSet.new
+        end
+
+        def self.reset_acquired(older_than_seconds)
```

Comment:
yes, they are alternative. In the next major version, I have been working on ways for these to co-exist (some in the db, some in memory), but not in version 0